readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -710,6 +710,14 @@ def test_wrong_inputs_in_webhookform(self):
         self.assertDictEqual(form.errors, {'url': ['Enter a valid URL.']})
         self.assertEqual(self.project.webhook_notifications.all().count(), 0)
 
+        data = {
+            'url': 'http://www.example.com/'
```

Comment:
We should add more tests cases here. Like urls that have http in the middle, end, don't start with a protocol, etc

Reply:
Will add more tests 👍